DatadogSessionReplay/Sources/Recorder/Recorder.swift

```diff
@@ -117,7 +119,7 @@ public class Recorder: Recording {
             return
         }
 
-        let touchSnapshot = recorderContext.touchPrivacy == .show ? touchSnapshotProducer.takeSnapshot(context: recorderContext) : nil
+        let touchSnapshot = touchSnapshotProducer.takeSnapshot(context: recorderContext)
```

Comment:
We don't check for the global touch privacy here anymore, it's now done in `WindowTouchSnapshotProducer`.